### DISCUSSION TITLE
null type

### DISCUSSION THREAD

**Discussion by giugallo** [victools/jsonschema-generator#345] [Q&A] [answered]
Is there a way to generate json-schema including type "null" for properties not required?
For example having a field like:
`private String myField;`

in the json-schema it becomes:
`"myField": {"type": "string"}`

I would to generate:
`"myField": {"type": ["string", "null"]}`

Is it possible?

**Comment by CarstenWickner** [comment, marked as answer]
Hi @giugallo,

Please enable the `Option.NULLABLE_FIELDS_BY_DEFAULT`.
That should give you the desired output.
